dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/apps/AppsResourceTest.java

```diff
@@ -459,15 +465,16 @@ public void Test_Create_App_Descriptor_Then_Create_App_Integration_Then_Delete_A
     }
 
     @Test
-    public void Test_Protected_Hidden_Secret_And_Values_Returned_Match_Descriptor() {
+    public void Test_Secret_Serializer_Returned_Values_Match_Descriptor_Verify_Hidden_Secrets_Are_Protected() throws Exception{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5358201)